test/kubemark/pre-existing/README.md

```diff
@@ -0,0 +1,45 @@
+# Kubemark Pre-existing Provider Guide
+
+**Pre-existing Kubernetes Cluster**
+- An already runnning Kubernetes cluster with master and minions.
+
+## Introduction
+
+When using the pre-existing provider, the Kubernetes cluster already exists.
+Therefore, the kubemark scripts will not create any infrastructure or a
+Kubernetes Cluster. Instead, it will use the resources provided by the existing
```

Comment:
It still creates a kubernetes cluster in some sense (a fake one) however doesn't create master. Maybe rather say `Therefore, the kubemark scripts will not create any infrastructure or start a kubemark master`.